examples/AN01034_using_the_device_control_library_over_usb/host/src/pause.h

```diff
@@ -1,7 +1,7 @@
-// Copyright 2016-2021 XMOS LIMITED.
+// Copyright 2016-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
-#ifndef __pause_h__
-#define __pause_h__
+#ifndef PAUSE_H_
```

Comment:
A single leading underscore is okay and appears frequently, for instance, in the header files in lib_xua.  As far as I'm aware, it's a matter of style whether to use one in a header guard or not.

Reply:
It seems that the usual way of adding header guards is the one I used: https://en.wikipedia.org/wiki/Include_guard.
This style is used in other XMOS repos, such as fwk_rtos. I would keep the most common style without a prepended underscore if nobody has any objection.

Reply:
That's fine with me.